lib/AppInfo/Application.php

```diff
@@ -38,6 +38,21 @@ public function __construct()
     public function register(IRegistrationContext $context): void
     {
         $context->registerAlternativeLogin(OIDCLoginOption::class);
+
+        // Try to get Files_External storage service
+        $context->registerService('storagesService', function($container) {
+            $storagesService = null;
+            try {
+                $storagesService = class_exists('\OCA\Files_External\Service\GlobalStoragesService') ?
+                    $container->query(\OCA\Files_External\Service\GlobalStoragesService::class) : null;
+            } catch (Exception $e) {}
+            return $storagesService;
```

Comment:
I'm assuming returning `null` here is fine

Reply:
Yes it is. Like before, the parameter `storagesService` in the constructor of a class would then be `null` as well.